Sources/Overload/OvUI/include/OvUI/Core/UIManager.h

```diff
@@ -133,5 +171,7 @@ namespace OvUI::Core
 		Modules::Canvas* m_currentCanvas = nullptr;
 		std::unordered_map<std::string, ImFont*> m_fonts;
 		std::string m_layoutSaveFilename = "imgui.ini";
+		const std::string m_defaultLayout;
+		const std::string m_layoutsPath;
```

Comment:
I'm thinking we should start using `std::filesystem::path` when storing path, as it's safer, and allow operations like:
```cpp
std::filesystem::path("path/to/something") / "folder" / "other_folder" / "file.txt"
```